### PR TITLE
initial version of the listener deepdive

### LINKED ISSUE #1985: feat: add deepdive about Argilla `listeners`
**Is your feature request related to a problem? Please describe.**
The listeners aren´t mentioned in the docs.

**Describe the solution you'd like**
N.A.

**Describe alternatives you've considered**
N.A.

**Additional context**
N.A.

### PR DESCRIPTION
**Pull request by davidberenstein1957**
# Description

I added a deepdive about listeners to our `features`section.

Closes #1985

**Type of change**

Please delete options that are not relevant.

- [X] Documentation update

### CODE REVIEW

File: docs/_source/guides/features/listeners.ipynb [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+   "id": "025f037c-bf3b-48f9-b347-e325737c07a0",
+   "metadata": {},
+   "source": [
+    "# Listeners\n",
+    "\n",
+    "This guide gives you a brief introduction to Argilla Listeners. Argilla Listeners enable you to build fine-grained complex workflows as background processes, like a low-key alternative to job scheduling directly integrated with Argilla. \n",
+    "\n",
+    "The main goal facilitate the user to define and customize their Argilla experience, and these Listeners are the back-bone of our [Argilla Plugins](https://github.com/argilla-io/argilla-plugins). Note that the tutorial about [active learning with small-text](../../tutorials/libraries/smalltext.md) is a great example of how powerful listeners can be. Alternatively, you can check the [Python Client](../../reference/python/python_listeners.rst) to get acquinted.\n",
+    "\n",
+    ".. nbinfo::\n",
```
Comment (dvsrepo) [inline docs/_source/guides/features/listeners.ipynb]: Use this instead so it's shown with the doc css styles:

<div class="alert alert-info">
</div>

File: docs/_source/guides/features/listeners.ipynb [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    "    rg.log(name, records) "
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "9a1a9ff5",
+   "metadata": {},
+   "source": [
+    "### start and stop listening"
```
Comment (dvsrepo) [inline docs/_source/guides/features/listeners.ipynb]:
```suggestion
    "### Start and stop listening"
```

File: docs/_source/guides/features/listeners.ipynb [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "025f037c-bf3b-48f9-b347-e325737c07a0",
+   "metadata": {},
+   "source": [
+    "# Listeners\n",
+    "\n",
+    "This guide gives you a brief introduction to Argilla Listeners. Argilla Listeners enable you to build fine-grained complex workflows as background processes, like a low-key alternative to job scheduling directly integrated with Argilla. \n",
+    "\n",
+    "The main goal facilitate the user to define and customize their Argilla experience, and these Listeners are the back-bone of our [Argilla Plugins](https://github.com/argilla-io/argilla-plugins). Note that the tutorial about [active learning with small-text](../../tutorials/libraries/smalltext.md) is a great example of how powerful listeners can be. Alternatively, you can check the [Python Client](../../reference/python/python_listeners.rst) to get acquinted.\n",
```
Comment (dvsrepo) [inline docs/_source/guides/features/listeners.ipynb]:
```suggestion
    "The main goal is to enable the user to define and customize their Argilla experience, and these Listeners are the back-bone of our [Argilla Plugins](https://github.com/argilla-io/argilla-plugins). Note that the tutorial about [active learning with small-text](../../tutorials/libraries/smalltext.md) is a great example of how powerful listeners can be. Alternatively, you can check the [Python Client](../../reference/python/python_listeners.rst) to get acquainted with the available options.\n",
```

File: docs/_source/guides/features/listeners.ipynb [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "9793adf5",
+   "metadata": {},
+   "source": [
+    "## Basics\n",
+    "\n",
+    "Listeners are decorators and wrap about a function that you would like to schedule. By defining a `query`, the `update_records` function gets two variables: 1) the `records` that we get from the dataset and query, 2) the `ctx` that contain function parameters like."
```
Comment (dvsrepo) [inline docs/_source/guides/features/listeners.ipynb]:
```suggestion
    "Listeners are decorators and wrap a function that you would like to schedule. By defining a `query`, the `update_records` function gets two variables: 1) the `records` that we get from the dataset and query, and 2) the `ctx` that contain function parameters like."
```
complete the last sentence: like...

**Review by dvsrepo — CHANGES_REQUESTED** [review]
Looking great!! 

Just some minor comments.

I'm unable to see the feature box here: https://docs.argilla.io/en/docs-add-deepdive-listeners/guides/features/features.html Maybe it's a cache issue on my side?

File: docs/_source/guides/features/listeners.ipynb (hunk 1 above)
Comment (davidberenstein1957) [reply docs/_source/guides/features/listeners.ipynb]: I think it is a cache, what should I use?

Comment (dvsrepo) [reply docs/_source/guides/features/listeners.ipynb]: ah sorry! I added the html snippet directly and it seems to remove it, now:

```html
<div class="alert alert-info">
</div>
```

**Pull request merged**